### PR TITLE
Add clipboard sending functionality, refs #12800

### PR DESCRIPTION
**Pull request by mcantelon**
Add the ability to, if the feature's enabled and configured, send the
contents of the clipboard to an external script for processing.

A new clipboard settings page has been added and new settings related
to clipboard sending.

A parent class for settings pages has been added.

### CODE REVIEW

File: apps/qubit/modules/user/actions/clipboardSendAction.class.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+      $this->redirect(array('module' => 'user', 'action' => 'clipboard'));
+    }
+    else
+    {
+      // Set message shown to user while message is sent
+      $sendMessageHtml = $this->context->i18n->__('%1%Sending...%2%', array('%1%' => '<h1>', '%2%' => '</h1>'));
+      $this->sendMessageHtml = sfConfig::get('app_clipboard_send_message_html', $sendMessageHtml);
+
+      // Set where and how data is to be sentl
```
Comment (sbreker) [inline apps/qubit/modules/user/actions/clipboardSendAction.class.php]: nitpick - typo!

**Review by sbreker — APPROVED** [review]
Looks good to me @mcantelon!  👍

File: apps/qubit/modules/user/actions/clipboardSendAction.class.php (hunk 1 above)
Comment (mcantelon) [reply apps/qubit/modules/user/actions/clipboardSendAction.class.php]: Hi Steve... where's the typo? I'm probably missing something obvious. ;)

Comment (sbreker) [reply apps/qubit/modules/user/actions/clipboardSendAction.class.php]: "sentl" right in the comment in the window above my first comment - line 48. I've approved the PR btw.  Looks great!

Comment (mcantelon) [reply apps/qubit/modules/user/actions/clipboardSendAction.class.php]: D'oh! Thanks! That light text fooled my eye. ;D

**Pull request merged**